pom.xml

```diff
@@ -288,8 +288,35 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+        <groupId>com.fasterxml.jackson.dataformat</groupId>
+        <artifactId>jackson-dataformat-yaml</artifactId>
+        <version>2.3.0</version>
+    </dependency>
+    <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-databind</artifactId>
+        <version>2.2.3</version>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-lang3</artifactId>
+        <version>3.4</version>
+    </dependency>
+    <dependency>
+      <groupId>org.example</groupId>
+      <artifactId>metadata-gen</artifactId>
+      <version>1.0</version>
+    </dependency>
   </dependencies>
 
+  <repositories>
+    <repository>
+        <id>local-maven-repo</id>
+        <url>file://${project.basedir}/lib</url>
```

Comment:
Please install metadata-gen to local repo by `mvn install` so you can use it without this section.
You can update CI to clone latest from metadata-gen repo, install in to local repo of CI and build plugin only after.

Reply:
I've used `mvn-install` plugin now. Do I also  need to also clone latest?